packages/pilot/src/containers/Refund/CardForm/index.js

```diff
@@ -130,15 +131,29 @@ class CardForm extends Component {
               </Col>
 
               <Col palm={12} tablet={5} desk={5} tv={5}>
-                <FormInput
-                  label={t('pages.refund.refund_amount_with_symbol')}
-                  name="amount"
-                  renderer={props => (
-                    <CurrencyInput
-                      {...props}
+                { isAuthorized
+                  ? (
+                    <Property
+                      title={t('pages.refund.refund_amount_with_symbol')}
+                      value={
+                        <span className={style.authorizedAmount}>
+                          {formatterCurrency(amount)}
+                        </span>
+                      }
                     />
-                  )}
-                />
+                  ) : (
```

Comment:
talvez mudar a identação aqui pra ficar no padrão que usamos ternario em outras situações matendo os simbolos `?` e `:` sempre no inicio da linha.
```suggestion
                  ) 
                  : (
```